static/js/components/LearningResourceCard.js

```diff
@@ -176,8 +180,9 @@ export function LearningResourceDisplay(props: Props) {
             <div className="row subtitles">
               {object.offered_by.length ? (
                 <Subtitle
-                  content={offeredBySearchLink(object.offered_by.join(", "))}
+                  content={formatOfferedBys(object.offered_by)}
                   label="Offered by - "
+                  htmlClass="offeror"
```

Comment:
Should there be a css definition for this class?

Reply:
I removed the class since you are correct, it is not in the definition

Reply:
@mbertrand